data/tag_windows.txt

```diff
@@ -1,74 +1,256 @@
+user_suspect_file
+  (data_type is 'fs:stat' OR data_type is 'fs:mactime:line') AND filename contains '/Users/' AND (filename contains '.exe' OR filename contains '.zip' OR filename contains '.rar' OR filename contains '.dll' OR filename contains '.cmd' OR filename contains '.rtf' OR filename contains '.pdf' OR filename contains '.doc' OR filename contains '.ppt' OR filename contains '.xls' OR filename contains '.vb' OR filename contains '.hta' OR filename contains '.jar' OR filename contains '.swf' OR filename contains '.reg' OR filename contains '.ps' OR filename contains '.ws' OR filename contains '.mht' OR filename contains '.ms' OR filename contains '.tmp' OR filename contains '.py' OR filename contains '.xhtm' OR filename contains '.js' OR filename contains '.bat' OR filename contains '.scr' OR filename contains '.sys' OR filename contains '.com' OR filename contains '.pif' OR filename contains '.inf' OR filename contains '.scf' OR filename contains '.chm' OR filename contains '.jnlp' OR filename contains '.gadget' OR filename contains '.appref-ms' OR filename contains '.application' OR filename contains '.sfx' OR filename contains '.lnk' OR filename contains '.otf' OR filename contains '.cpl' OR filename contains '.cab')
```

Comment:
`filename contains '/Users/'` this is going to break now with pyfsntfs being the default NTFS back-end.

Reply:
Can you explain me, the change in structure, filname field will not exist?

Reply:
sry, good point forgot to relay that detail ;) 

the NTFS dfVFS back-end uses backslash (\), to be as close to the original as possible, instead of forward slash (/) which is used by TSK for everything. I'll need to re-assess how to do this in a less fragile way, but that is something a later point in time.

Reply:
Ok, it may impact other rules then present in tag_windows.txt ...